src/Clients/CaradhrasMainClient.php

```diff
@@ -25,15 +68,73 @@ public function getTransfer(array $filters): P2PTransferPayload
     }
 
     /**
-     * @param  int  $accountId
-     * @param  AccountStatus  $status
+     * Get individual.
+     *
+     * @param  string  $registrationId
+     * @param  string  $document
      * @return object
+     * @throws Idez\Caradhras\Exceptions\CaradhrasException
      */
-    public function cancelAccount(int $accountId, AccountStatus $status = AccountStatus::Canceled): object
+    public function findIndividual(string $registrationId, string $document): object
+    {
+        $response = $this
+            ->apiClient(throwsHttpError: false)
+            ->asJson()
+            ->get("/v2/individuals", ['document' => $document]);
+
+        if ($response->failed()) {
+            $message = $response->status() === 404 ? 'Failed to get individual.' : 'Individual not found.';
+            $statusCode = $response->status() === 404 ? 404 : 502;
+
+            throw new CaradhrasException($message, $statusCode);
```

Comment:
Poderia ser algo como veja na API o `ApprovalException`
```suggestion
            throw new CaradhrasException::failedGetIndividual()
            
            // or 
            
            throw new CaradhrasException::IndividualNotFound()
```